src/CMakeLists.txt

```diff
@@ -409,6 +429,44 @@ if(MSVC)
 		OpenAL::OpenAL
 		LibLZMA::LibLZMA
 	)
+elseif(ANDROID)
+	target_include_directories(${PROJECT_NAME}
```

Comment:
Here we have to define all those common libraries and includes into a variable to not repeat for every new platform.